zipline/finance/blotter.py

```diff
@@ -179,64 +185,62 @@ def process_benchmark(self, benchmark_event):
         return
         yield
 
-    def process_trade(self, trade_event):
+    def process_open_orders(self):
+        """
+        Creates a list of transactions based on the current open orders,
+        slippage model, and commission model.
+
+        Parameters
+        ---------
+        None
+
+        Notes
+        -----
+        This method book-keeps the blotter's open_orders dictionary, so that
+         it is accurate by the time we're done processing open orders.
+
+        Returns
+        -------
+        A list of transactions resulting from the current open orders.  If
+        there were no open orders, an empty list is returned.
+        """
+        closed_orders = []
+        transactions = []
+
+        for asset, asset_orders in self.open_orders.iteritems():
+            for order, txn in self.slippage_func(asset_orders, self.current_dt):
+                direction = math.copysign(1, txn.amount)
+                per_share, total_commission = self.commission.calculate(txn)
+                txn.price += per_share * direction
+                txn.commission = total_commission
+                order.filled += txn.amount
 
-        if trade_event.sid not in self.open_orders:
-            return
+                if txn.commission is not None:
+                    order.commission = (order.commission or 0.0) + \
+                                       txn.commission
 
-        if trade_event.volume < 1:
-            # there are zero volume trade_events bc some stocks trade
-            # less frequently than once per minute.
-            return
+                txn.dt = pd.Timestamp(txn.dt, tz='UTC')
+                order.dt = txn.dt
```

Comment:
Do we currently set the `dt` of the order to the txn `dt`. My understanding is that those are two distinct moments.